gaeguli/gaeguli-internal.h

```diff
@@ -29,6 +29,9 @@
 #define GAEGULI_PIPELINE_VSRC_STR       "\
         %s ! capsfilter name=caps ! %s ! tee name=tee allow-not-linked=1 "
 
+#define GAEGULI_PIPELINE_IMAGE_STR    "\
+        valve name=valve drop=1 ! jpegenc ! multifilesink next-file=0 async=0 location=/tmp/%%05d.jpg"
```

Comment:
We can't assume that `/tmp` is always available. I think it is better to use an absolute path to let the programmer take the responsiblity.

Reply:
Uploaded a new revision that asynchronously returns JPEG image data wrapped into `GBytes`.